apps/ex_fleet_yards_import/mix.exs

```diff
@@ -26,7 +26,8 @@ defmodule ExFleetYardsImport.MixProject do
   # Run "mix help deps" to learn about dependencies.
   defp deps do
     [
-      {:ex_fleet_yards, in_umbrella: true}
+      {:ex_fleet_yards, in_umbrella: true},
+      {:neuron, "~> 5.0.0"}
```

Comment:
needs an update to the `nix/mix.nix` file to not break nix builds.

Reply:
is there a task to regenerate the mix.nix?

Reply:
there is a task, but currently seems broken, `mix2nix > nix/mix.nix` is how I generated it (having mix2nix installed via the devenv)